README.md

```diff
@@ -1,75 +1,171 @@
-- Node package. [Link to download node](https://nodejs.org/en/download/)
+### Prerequisites
 
-## To Install type the following command
+In order to run this project you need:
+
+- Basic Knowledge of HTML and CSS
+- Basic Understanding of Linters
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- git clone https://github.com/SiveMdluli/Hello-Portfolio.git
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/SiveMdluli/Hello-Portfolio.git
+```
+--->
+
+### Install
+
+Install this project with:
 
 - npm install
-- sudo nmp install
 
-## To Run App type the following command in your terminal
 
-  - npm start
 
-### To Run test cases type the following command
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "**/*.{css,scss}"
+- npx hint .
+
+### Deployment
 
-  - npm test
+You can deploy this project using:
 
-## Author
+- Github tools
 
-👤 Khuraijam Henary Singh
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [SiveMdluli](https://github.com/SiveMdluli)
+- Twitter: [@SiveMdluli1](https://twitter.com/SiveMdluli1)
+- LinkedIn: [Sive-Mdluli](https://linkedin.com/in/Sive-Mdluli)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- GitHub: [@sivemdluli](https://github.com/sivemdluli)
-- Twitter: [@sivemdluli](https://twitter.com/sivemdluli1)
-- LinkedIn: [sive-mdluli](https://www.linkedin.com/in/sive-mdluli)
 
 ## 🔭 Future Features <a name="future-features"></a>
 
- [To Add more work pages and styles and make it a desktop version too]
+> Describe 1 - 3 features you will add to the project.
+
+- Add contact page.
+- Add Project's section
 
-## 🤝 Contributing
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues).
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project give this repository a 5 star rating.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+- Microverse Review Team
+- My Coding Partner
+- I, Sive and my Family
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Show your support
 
-Give a [⭐️](../../stargazers) if you like this project!
+## 📝 License <a name="license"></a>
 
+This project is [MIT](./LICENSE) licensed.
 
-## Acknowledgments
-- Microverse
-- All my fellow micronauts
-- My Family
 
```

Comment:
- [ ] `index.html`: Kindly make sure you stick as much as possible to you Figma chose template as required for the project. Below you can see how looks you project on some mibile widths
![Screenshot from 2023-02-22 18-00-58](https://user-images.githubusercontent.com/83042742/220683494-e19f46eb-8541-4d78-a420-91bab903772e.png)

Reply:
@tresorsawasawa  Can you please take a look, I have made changes to the hamburger icon. Also fixed new linter errors.